AmdExcludedFunctionParameterCountCheck.js

```diff
@@ -0,0 +1,22 @@
+require(
+	['jquery', 'mw', 'wikia.window', 'wikia.tracker', 'wikia.geo'],
+	function ($, mw, context, tracker, geo) {}
+);
+
+define(
+	'wikia.awesomeModule',
+	['jquery', 'mw', 'wikia.window', 'wikia.tracker', 'wikia.geo'],
+	function ($, mw, context, tracker, geo) {}
+);
+
+function smallCompliantFunction(a, b, c, d) {}
+
+function bigEvilFunction(a, b, c, d, e) {} // Noncompliant
+
+var o = {
+	smallCompliantMethod: function (a, b, c, d) {},
+	bigEvilMethod: function (a, b, c, d, e) {} // Noncompliant
+};
+
+someCompliantAsyncThing.doStuffWithCallback(function (a, b, c, d) {});
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "a". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "b". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "c". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "d". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)